.github/workflows/scripts/e2e/gmc_gaudi_test.sh

```diff
@@ -87,7 +87,7 @@ function validate_chatqna() {
    sleep 120
 
    kubectl get pods -n $APP_NAMESPACE
-   # send request to chatqnA
+   # send request to chatqna
    export CLIENT_POD=$(kubectl get pod -n $APP_NAMESPACE -l app=client-test -o jsonpath={.items..metadata.name})
```

Comment:
Hi @gyohuangxin, if so, I suggest that add a new namespace named `CHATQNA_NAMESPACE` and it should be `CHATQNA_NAMESPACE="${APP_NAMESPACE}-chatqna"`

Reply:
And the same as in `gmc_xeon_test.sh`

Reply:
@zhlsunshine Thanks for your suggestion, and I've add the new namespace "CHATQNA_NAMESPACE="${APP_NAMESPACE}-chatqna". 
Besides, this [line of code](https://github.com/opea-project/GenAIInfra/blob/main/.github/workflows/scripts/e2e/gmc_xeon_test.sh#L292) will replace namespace "default" with `APP_NAMESPACE`, but the `APP_NAMESPACE` won't exist after this change, what namespace should it be changed to?

Reply:
Hi @gyohuangxin, the GMC controller would handle this and it depends on every example. You also can refer to the comment [here](https://github.com/opea-project/GenAIInfra/pull/142#issuecomment-2199191310)